content/blog/2017/2017-03-06-jenkins-community-survey.adoc

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "The State of Jenkins - 2016 Community Survey "
+tags:
+- community
+- survey
+- continuous delivery
+author: brao
+---
+
+[NOTE]
+====
+This is a guest post by Bhavani Rao, Marketing Manager at link:http://cloudbees.com[CloudBees] 
+====
+
+Last fall, prior to Jenkins World, CloudBees conducted a link:https://jenkins.io/blog/2016/09/09/take-the-2016-jenkins-survey-blog/[Community Survey on behalf of the Jenkins community]. We received over 1200 responses  – and thanks to this input, we 
```

Comment:
Nit: We prefer host relative URLs (those starting with `/blog` in this case) to facilitate local development.